extensions/database.py

```
@@ -3,54 +3,46 @@
 #
 
 # HANDLER -> LOAD
-import os
-import sys
-sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
-from handler import *
+from website import *
```

Ruff (0.11.9)

`from website import *` used; unable to detect undefined names

(F403)